.buildconfig-android.yml

```diff
@@ -1,58 +1,58 @@
 libraryVersion: 0.31.2
 groupId: org.mozilla.appservices
 projects:
-  fxa-client-library:
+  fxaclient:
     path: components/fxa-client/android
     artifactId: fxaclient
-    publishedArtifacts: [fxaclient, fxaclient-forUnitTests, fxaclient-withoutLib]
+    publishedArtifacts: [fxaclient]
     description: Firefox Accounts client.
-  logins-library:
+  logins:
     path: components/logins/android
     artifactId: logins
-    publishedArtifacts: [logins, logins-forUnitTests, logins-withoutLib]
+    publishedArtifacts: [logins]
     description: Sync 1.5 logins implementation.
-  places-library:
+  places:
     path: components/places/android
     artifactId: places
-    publishedArtifacts: [places, places-forUnitTests, places-withoutLib]
+    publishedArtifacts: [places]
     description: Low level places storage implementation.
-  rustlog-library:
+  rustlog:
     path: components/rc_log/android
     artifactId: rustlog
-    publishedArtifacts: [rustlog, rustlog-forUnitTests, rustlog-withoutLib]
+    publishedArtifacts: [rustlog]
     description: Android hook into the log crate.
-  viaduct-library:
+  httpconfig:
     path: components/viaduct/android
     artifactId: httpconfig
-    publishedArtifacts: [httpconfig, httpconfig-forUnitTests, httpconfig-withoutLib]
+    publishedArtifacts: [httpconfig]
     description: Component allowing the configuration of Rust HTTP stack.
-  as-support-library:
+  native-support:
```

Comment:
This is a breaking change for a-c, which takes a dependency on this project as "org.mozilla.appservices:support" for testing purposes here:

  https://github.com/mozilla-mobile/android-components/blob/master/components/browser/storage-sync/build.gradle#L67

I don't know why that dependency is required though, and maybe it can go away as a result of this patch. I'm going to try removing it and see what happens.

In any case, it's probably worth calling this out as a breaking change in the changelog.

Reply:
> I don't know why that dependency is required though, and maybe it can go away as a result of this patch.
> I'm going to try removing it and see what happens.

Whoop, I see you already did this in your a-c branch, I just had the wrong thing checked out by accident.